content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -9,61 +9,71 @@ weight: 20
 ---
 
 <!-- overview -->
-This page describes common concepts in the Kubernetes API.
-
+This page describes Kubernetes' API and conventions for interacting with it.
```

Comment:
Why this revision?

Reply:
Concept documentation doesn't belong in the API reference section.
Should we:
 - focus on the reference nature and make the tone of the page match its current home
 - migrate the page into the concepts section and add redirects
?

This change adapts the page to its current home.

Reply:
- I think it would help to retitle this PR.
- I will read through the changes, but I think the word "concepts" is used generically here (similar to conventions).
You could remove the original sentence and not add a replacement sentence. The following two sentences are long and could possibly be broken up.
- Also, the concepts section has a related page, `concepts/overview/kubernetes-api.md` and then there is the page,
`reference/using-api/_index.md` (`This section provides reference information for the Kubernetes API.`)

Reply:
> the concepts section has a related page, `concepts/overview/kubernetes-api.md` and then there is the page,
`reference/using-api/_index.md`
> (This section provides reference information for the Kubernetes API.)

I'm fairly convinced that having this information spread across two pages, with the current split, is unhelpful to readers.

Reply:
OK. The concept section has enough information to introduce the k8s APIs and provide links to more information. The k8s API concepts overview page provides that (https://kubernetes.io/docs/concepts/overview/kubernetes-api/).
One section that could possibly move to the reference section: https://kubernetes.io/docs/concepts/overview/kubernetes-api/#api-specification